File: SEproject/Data/Enrollment.cs
using System;
using System.Collections.Generic;

namespace SEproject.Data
{
    public partial class Enrollment
    {
        public string StudentId { get; set; } = null!;
        public string CourseId { get; set; } = null!;

        public virtual Course Course { get; set; } = null!;
        public virtual Student Student { get; set; } = null!;
    }
}

File: SEproject/ClientApp/src/app/data/models/enrollment.ts
import { Student } from "../../models/student";
import { Course } from "./course";

export class Enrollment {
    courseID: string;
    studentID: string;
    student: Student;
    course: Course;

  constructor(courseID: string, studentID: string, student: Student, course: Course) {
        this.courseID = courseID;
        this.studentID = studentID;
        this.student = student;
        this.course = course;
    }
}
